Lib/test/test_fractions.py

```diff
@@ -1530,6 +1530,8 @@ def test_format_f_presentation_type(self):
             (F(51, 1000), '.1f', '0.1'),
             (F(149, 1000), '.1f', '0.1'),
             (F(151, 1000), '.1f', '0.2'),
+            (F(22, 7), '.02f', '3.14'),  # issue gh-130662
+            (F(22, 7), '005.02f', '03.14'),
```

Comment:
Because of this, we should add a NEWS entry (it's still a change of behavior, albeit minor).